actions-cleanup-openstack-vms/readme.md

```diff
@@ -0,0 +1,28 @@
+# Overview
+
+This action is clean up VMs that are older than a certain period of time in openstack
+
+# Example
+
+```yaml
+name: "clean up idle VMs in OVH openstack"
```

Comment:
Here we talk about "idle". We need to be clear if it's idle or old.

Reply:
What i mean here is that the VMs are older. maybe they have ran a job and have not been deleted properly.